src/Driver/Driver.jl

```diff
@@ -381,69 +384,39 @@ function invoke!(
             GenericCallbacks.EveryXSimulationSteps(Settings.monitor_courant_interval) do (
                 init = false
             )
-                simtime = ODESolvers.gettime(solver)
-                Δt = solver_config.dt
-                c_v = DGmethods.courant(
-                    nondiffusive_courant,
-                    solver_config;
-                    direction = VerticalDirection(),
-                )
-                c_h = DGmethods.courant(
-                    nondiffusive_courant,
-                    solver_config;
-                    direction = HorizontalDirection(),
-                )
-                ca_v = DGmethods.courant(
-                    advective_courant,
-                    solver_config;
-                    direction = VerticalDirection(),
-                )
-                ca_h = DGmethods.courant(
-                    advective_courant,
-                    solver_config;
-                    direction = HorizontalDirection(),
-                )
-                cd_v = DGmethods.courant(
-                    diffusive_courant,
-                    solver_config;
-                    direction = VerticalDirection(),
-                )
-                cd_h = DGmethods.courant(
-                    diffusive_courant,
-                    solver_config;
-                    direction = HorizontalDirection(),
-                )
-                @info @sprintf """
-                ================================================
-                Courant numbers at simtime: %8.2f
-                Δt = %8.2f s
-
-                ------------------------------------------------
-                Acoustic (vertical) Courant number    = %.2g
-                Acoustic (horizontal) Courant number  = %.2g
-                ------------------------------------------------
-                Advection (vertical) Courant number   = %.2g
-                Advection (horizontal) Courant number = %.2g
-                ------------------------------------------------
-                Diffusion (vertical) Courant number   = %.2g
-                Diffusion (horizontal) Courant number = %.2g
-                ================================================
-                """ simtime Δt c_v c_h ca_v ca_h cd_v cd_h
-                return nothing
+                show_courant(solver_config)
+                nothing
             end
         callbacks = (callbacks..., cbcfl)
     end
 
+    if Settings.checkpoint_walltime > 0
+        step = Ref(1)
+        cbcheckpoint = GenericCallbacks.EveryXWallTimeSeconds(
```

Comment:
Could we enable the option to use `SimulationTime` callbacks here ?

Reply:
Since that callback implementation is commented, I think it needs tests when we re-enable it. I'll do that in a separate PR.

Reply:
Opened #909 to track this.